PSReadLine/Completion.cs

```diff
@@ -757,7 +761,7 @@ private void PossibleCompletionsImpl(CommandCompletion completions, bool menuSel
                                 _mark = savedUserMark;
                             }
                             // without render all key chords that just move cursor leave selection visible, but it can be wrong
-                            Render();
+                            if (!undo && !keepSelection) Render();
```

Comment:
I'm not always consistent about single line if statements, but in this case, I'd prefer braces and the body not on the same line as the condition.